just_audio/android/build.gradle

```diff
@@ -48,11 +48,10 @@ android {
     }
 
     dependencies {
-        def exoplayer_version = "2.18.7"
-        implementation "com.google.android.exoplayer:exoplayer-core:$exoplayer_version"
-        implementation "com.google.android.exoplayer:exoplayer-dash:$exoplayer_version"
-        implementation "com.google.android.exoplayer:exoplayer-hls:$exoplayer_version"
-        implementation "com.google.android.exoplayer:exoplayer-smoothstreaming:$exoplayer_version"
+        def exoplayer_version = "1.0.2"
```

Comment:
please consider using the latest [1.4.1 version of media3 exo player](https://github.com/androidx/media/releases)

Reply:
Thank you for the feedback. And good point, I had a quick look at using the latest version and that is not so trivial unfortunately. It would be easy to switch to version 1.1.1 but anything newer would require several code changes. The class _ConcatenatingMediaSource_ was deprecated and removed from the media3 library. I will have a look at what changes are required to jump to the latest version. But it might also be worth considering to split it in two separate pull requests.

Reply:
I'd like to say we can just do an intermediate release with version 1.1.1 and then work on 1.4.1, but  video_player uses 1.4.1, and we have seen version conflicts occur before in apps that use both plugins when the exoplayer dependencies aren't in sync.

Let me take a look at this and see if I can tackle 1.4.1 using this PR as a starting point...